pages/dkp/kommander/2.1/install/air-gapped/index.md

```diff
@@ -127,9 +127,7 @@ export VERSION=v2.1.1
     wget "https://downloads.d2iq.com/kommander/airgapped/${VERSION}/kommander_image_bundle_${VERSION}_linux_amd64.tar" -O kommander-image-bundle.tar
     ```
 
-    <p class="message--note"><strong>NOTE: </strong>This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the <a href="https://www.gnu.org/licenses/agpl-3.0.en.html">GNU Affero General Public License 3.0</a>. Complete source code for MinIO is available <a href="https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z">here</a> and <a href="https://github.com/minio/minio/tree/RELEASE.2021-07-30T00-02-00Z">here</a>.</p>
-
-1.  Place the bundle in a location where you can load and push the images to your private Docker registry.
+1.  Unpack the bundle and the nested bundle within in a location where you can load and push the images to your private Docker registry.
```

Comment:
This needs to be reviewed by whomever is packaging the new files.      If I understand correctly, you should NOT unpack twice but just once, the script below still takes a .tar file.

Also, I think this should be more explicit.  I think we should give the exact steps the user should execute here so there is no confusion; 'unbundle' is not specific enough.